spring-boot-modules/spring-boot-properties-2/src/main/java/com/baeldung/properties/value/defaults/ValuesWithDefaultsApp.java

```diff
@@ -60,10 +62,10 @@ public void afterInitialize() {
     	
     	// arrays
```

Comment:
What's the purpose of introducing these changes, and how are they relevant to current task?

Reply:
I did some improvements in the code with the suggestion of my SonarLint.

Reply:
Ok, that's good, but let's revert these.
We may have a separate issue with performance improvements.

Reply:
Ok, I will revert but I don't understand why we follow this approach, having "separate issue", I always follow this principle: https://stepsize.com/blog/how-to-be-an-effective-boy-girl-scout-engineer.
The change doesn't affect the behavior of our code.

Reply:
Done I revert that changes. I also added a commons-lang because was needed in that class map primitive arrays to Integer objects `ArrayUtils.toObject()`